### DISCUSSION TITLE
无法运行

### DISCUSSION THREAD

**Discussion by X-712** [ClassIsland/ClassIsland#61] [Q&A] [answered]
下载压缩包解压以后只有exe文件，双击后点击运行没有反应，右键以管理员身份运行也没有反应

**Comment by HelloWRC** [comment, marked as answer]
您好，请先检查您的设备是否满足 [README](https://github.com/HelloWRC/ClassIsland?tab=readme-ov-file#%E5%BC%80%E5%A7%8B%E4%BD%BF%E7%94%A8) 所列举的要求。如果您的设备满足要求，请再次尝试运行，并留意一下在运行过程中有没有弹出错误提示。

**Reply by X-712** [reply]
都满足的（Aero效果就是个性化里那个启用透明效果对吧），但是运行后就没有反应，错误提示也没有

**Reply by HelloWRC** [reply]
您可以尝试再运行一下，然后打开Windows事件查看器，然后在“Windows 日志”->“应用程序”条目下看看有没有与ClassIsland相关的错误日志。如果有，请把相关的日志条目贴上来。

**Reply by X-712** [reply]
Message: Failed to load the dll from [C:\Program Files\dotnet\host\fxr\8.0.3\hostfxr.dll], HRESULT: 0x80070057
The library hostfxr.dll was found, but loading it from C:\Program Files\dotnet\host\fxr\8.0.3\hostfxr.dll failed
  - Installing .NET prerequisites might help resolve this problem.
     https://go.microsoft.com/fwlink/?linkid=798306

**Reply by X-712** [reply]
这个网址点进去不知道该下载哪个，系统是win7

**Reply by HelloWRC** [reply]
根据文档，您还需要安装以下内容，请根据您的系统架构选择安装：

- Microsoft Visual C++ 2015-2019 Redistributable] [64-bit](https://aka.ms/vs/16/release/vc_redist.x64.exe) / [32-bit](https://aka.ms/vs/16/release/vc_redist.x86.exe)
- KB3063858 [64-bit](https://www.microsoft.com/download/details.aspx?id=47442) / [32-bit](https://www.microsoft.com/download/details.aspx?id=47409)

> 来源：https://learn.microsoft.com/zh-cn/dotnet/core/install/windows?tabs=net80#additional-deps
